.eslintrc.js

```diff
@@ -50,6 +50,8 @@ module.exports = {
           {
             allowNumber: true,
             allowBoolean: true,
+            allowAny: true,
+            allowNullish: true,
```

Comment:
At this point we can just disable this rule. I agree it is somewhat annoying, but it does very well prevent you from getting logs like `actual value: [Object object], expectedValue: undefined`

Reply:
ok, so I'll remove the allowAny (so you should either toString() or JSON.stringify) - but adding "eslint-ignore" before almost every debug line is making the exception-case too common...

Reply:
ok, so will you?